lib/Backends/CPU/CPUDeviceManager.cpp

```diff
@@ -116,8 +121,10 @@ void CPUDeviceManager::evictNetworkImpl(std::string functionName,
                                         EvictFunctionCBTy evictCB) {
   DCHECK(evictCB != nullptr);
 
-  if (functions_.erase(functionName)) {
-    usedMemoryBytes_ -= functionCost_; // TODO: static moduleSize
+  auto it = functions_.find(functionName);
+  if (it != functions_.end()) {
+    usedMemoryBytes_ -= it->second->getRuntimeBundle().getConstantWeightSize();
```

Comment:
If we use all functions above and subtract per single function here can we get out of sync?

Reply:
`evictNetwork` only allows you to specify the name of one function to evict. So I don't see any way to avoid getting "out of sync".